src/libcprover-rust/other/example.c

```diff
@@ -0,0 +1,7 @@
+// File just to allow smoke testing of Rust API while tests are run.
```

Comment:
Shouldn't test code be in the `regression` or `unit` directories? `other` seems like a particularly unhelpful categorisation.

Reply:
I agree that `other/` seems a bit misnamed (will probably change to something else as part of another PR), but I don't think this code should be in either `regression/` or `unit/` folders, on the basis that this is a small smoke test that is a dependency of the testing of the Rust API. 

All of the testing of the Rust API is happening inside the Rust project, and needs to be independent (as much as it can) and have all of its dependencies around so that a testing run is logically coherent and maintainable.

Reply:
But then we could have a `test` folder alongside `src`, couldn't we?